lib/vast/Frontend/Options.cpp

```diff
@@ -49,7 +49,6 @@ namespace vast::cc {
     std::optional< string_ref > vast_args::get_option(string_ref name) const {
         if (auto opt = detail::get_option_impl(args, name)) {
             if (auto [lhs, rhs] = opt->split('='); !rhs.empty()) {
-                VAST_ASSERT(!is_options_list(rhs));
```

Comment:
why did you removed this? if you want to use list option you should pick it using `get_options_list`

Reply:
Because for `option="pass1;pass2"` and call to `get_options_list` this will assert on `pass1;pass2` as this is used to parse `[lhs,rhs]`.